packages/docs-reanimated/docs/fundamentals/getting-started.mdx

```diff
@@ -91,7 +91,32 @@ To learn more about the plugin head onto to [Reanimated babel plugin](/docs/fund
 
 </details>
 
-### Step 3: Clear Metro bundler cache (recommended)
+### Step 3: Wrap metro config with reanimated wrapper (recommended)
```

Comment:
I strongly advocate for **not** adding installation steps to Reanimated in the fundamentals 

As this metro config is not required I would omit it from the fundamentals/getting-started

We could ask Expo to include it in their metro config tho

Reply:
Could you elaborate? We find this feature to be of substantial benefit to the user, since eventually they would be guided to their offending code in errors from Reanimated.

I also don't like the fact that it bloats the installation procedure and makes it less approachable. How about creating a new subpage (albeit close to `getting-started`) about recommended installation steps/good practices?

Reply:
I can agree with @tjzel that users will benefit from adding this metro config wrapper and we should somehow make it clear for users that such a wrapper exists. Can you suggest any better solution than the current one? I like the idea that @tjzel suggested to add separate page with recommended configuration steps if you don't want to include this step in the getting started page.